ocp.tf

```diff
@@ -16,6 +16,7 @@ locals {
   # Generates cluster_id as combination of cluster_id_prefix + (random_id or user-defined cluster_id)
   cluster_id   = var.cluster_id == "" ? random_id.label[0].hex : (var.cluster_id_prefix == "" ? var.cluster_id : "${var.cluster_id_prefix}-${var.cluster_id}")
   name_prefix  = var.use_zone_info_for_names ? "${local.cluster_id}-${var.ibmcloud_zone}" : local.cluster_id
```

Comment:
Also add a `-` at the end in similar format to `node-prefix`

Reply:
I'll make it consistent with node_prefix.